src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -46,6 +46,8 @@ internal sealed partial class ControlFlowGraphBuilder : OperationVisitor<int?, I
         private IOperation? _forToLoopBinaryOperatorRightOperand;
         private IOperation? _currentAggregationGroup;
         private bool _forceImplicit; // Force all rewritten nodes to be marked as implicit regardless of their original state.
+        private PooledDictionary<IOperation, IOperation>? _placeholderDictionary;
```

Comment:
>IOperation [](http://example.com/codeflow?start=33&length=10)

Is this always IInterpolatedStringHandlerArgumentPlaceholderOperation?

Reply:
This is gone now.